api/src/pcapi/core/educational/models.py

```diff
@@ -177,6 +178,12 @@ def hasBookingLimitDatetimesPassed(cls):  # pylint: disable=no-self-argument
             .where(CollectiveStock.hasBookingLimitDatetimePassed.is_(True))
         )
 
+    @property
+    def subcategory(self) -> subcategories.Subcategory:
+        if self.subcategoryId not in subcategories.ALL_SUBCATEGORIES_DICT:
+            raise ValueError(f"Unexpected subcategoryId '{self.subcategoryId}' for offer {self.id}")
```

Comment:
for collective offer ? sinon si on voit l'erreur passer on risque de regarder dans le mauvais modèle non ?

Reply:
note: c'est le wording qui est pas bon